ClamavPlugin.php

```diff
@@ -115,7 +104,8 @@ function getActions($request, $verb) {
 	function manage($args, $request) {
 		switch ($request->getUserVar('verb')) {
 			case 'settings':
-				$contextID = (!is_null($this->_getBackwardsCompatibleContext()) ? $this->_getBackwardsCompatibleContext()->getId() : PKPApplication::CONTEXT_SITE);
+				$context = Application::get()->getRequest()->getContext();
+				$contextID = $context->getId();
```

Comment:
There is some risk to this change.

Previously, we checked to see if `getContext()` returned `null`, and if so, substituted `PKPApplication::CONTEXT_SITE` for the context id.  The current code presumes that `$context` must not be null.

This can be checked by trying to manage the plugin from the Site level, rather than from the journal level. Or, we could re-introduce the `if` statement presumptively.

Reply:
#23